### PR TITLE
update axios npm version w.r.to npm audit fix

### PR DESCRIPTION
**Pull request by makambalaji**
### What does this PR do?
This PR fixes the npm vulnerabilities [Cherry pick of https://github.com/eclipse/che/pull/21046]

### How to test this PR?

Execute the Happy flow tests

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix or new feature ](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix-or-new-feature)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: tests/e2e/package-lock.json [hunk 1]
```diff
@@ -5,9 +5,9 @@
   "requires": true,
   "dependencies": {
     "@eclipse-che/api": {
-      "version": "7.40.1",
-      "resolved": "https://registry.npmjs.org/@eclipse-che/api/-/api-7.40.1.tgz",
-      "integrity": "sha512-2ij3D79RMcbV88XKbhI2fK3RgFex+qXDod4ZzXAx6LXoLHIdGTlYBs0EiCOmyuTYe2h4uB1tixRzGIVDTgc7PA=="
+      "version": "7.42.0",
```
Comment (dmytro-ndp) [inline tests/e2e/package-lock.json]: Let's keep 7.40.1 in here.

Comment (makambalaji) [reply tests/e2e/package-lock.json]: It creates the conflict between package.json and pacakage-lock.json. Is it ok to modify the @eclipse-che/api version from `latest` to `7.40.1` in package.json file

Comment (dmytro-ndp) [reply tests/e2e/package-lock.json]: yes, please

Comment (makambalaji) [reply tests/e2e/package-lock.json]: modified the package-lock.json file as follows
```
    "version": "7.40.1",
      "resolved": "https://registry.npmjs.org/@eclipse-che/api/-/api-7.40.1.tgz",
      "integrity": "sha512-2ij3D79RMcbV88XKbhI2fK3RgFex+qXDod4ZzXAx6LXoLHIdGTlYBs0EiCOmyuTYe2h4uB1tixRzGIVDTgc7PA=="
```
Executed the command `npm ci`, looks like no issue, so pushing the same changes

Comment (dmytro-ndp) [reply tests/e2e/package-lock.json]: There is still `"@eclipse-che/api": "latest"` in package.json.

**Review by dmytro-ndp — APPROVED** [review]
LGTM

**Pull request merged**